update/update-version.md

```diff
@@ -1,139 +1,187 @@
-        - Click on copy button to copy the Workspace ID.
+        Replace `<API-KEY>` with the value of your API key.
 
-    1. Get Workspace ID from Schematics Workspace: Alternatively, if not using Projects, but using Schematics directly do the following...
-        - Go to [Schematics Workspaces]( https://cloud.ibm.com/schematics/workspaces) and select the location from the dropdown that the workspace is in.
-        - Select the workspace associated with your vsi deployment.
-        - Click **Settings** on the left hand side Menu.
-        - Copy **Workspace ID** from the Details tab.
+    1. Find your Schematics workspace ID:
+        - If you are using IBM Cloud Projects:
+            1. Go to [IBM Cloud Projects](https://cloud.ibm.com/projects)
+            1. Select the project that is associated with your VSI deployment.
+            1. Click the **Configurations** tab.
+            1. Select the configuration that is associated with your VSI deployment.
+            1. Find the **Workspace** ID for the project toward the right side of the screen.
+            1. Copy the ID.
 
-    ```sh
-    export WORKSPACE_ID="<workspace-id>"
-    ```
+        - If you are not using IBM Cloud Projects:
+            1. Go to [Schematics Workspaces](https://cloud.ibm.com/schematics/workspaces)
+            1. Select the location that the workspace is in.
+            1. Select the workspace associated with your VSI deployment.
+            1. Click **Settings**.
+            1. Copy the **Workspace ID**.
+
+    1. Run the following command to set the workspace ID as an environment variable:
+
+        ```sh
+        export WORKSPACE_ID="<workspace-id>"
+        ```
+
+1. Download the script by running this Curl command:
 
-1. Download the script to your local machine.
     ```sh
     curl https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/main/update/schematics_update_v3.x.x_to_v4.x.x.sh > schematics_update_v3.x.x_to_v4.x.x.sh
     ```
 
-1. Retrieve the VPC IDs and region of any VPCs in which VSIs are deployed.
-    1. Get VPC ID from IBM Cloud UI Console:
-        - In the left navigation column, click on **VPC Infrastructure** > **VPCs**.
-        - Select the region in which the VPC of the VSI was deployed.
-        - Select the VPC associated with the VSI deployment.
-        - Copy the **VPC ID** from the Overview tab.
-
-1. [Optional] If the VSIs are being deployed in an account that is different from the account where the Schematics Workspace are located, you need to provide an IBM Cloud API key, which can be used for fetching the VPC details. In this case, you can pass the VPC API-KEY to the script using `-k` flag. For more information, see https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui
+1. Get the VPC IDs and the regions of any VPCs that the VSIs are deployed in by using the IBM Cloud console:
```

Comment:
perhaps mention that this needs to be done in the account where the VSIs exist (which may be different to the account in which the projects / schematics is)